web/src/lib/types/types.ts

```diff
@@ -152,8 +152,10 @@ export type Funding = {
   urlSlug: string;
   callToActionLink: string;
   callToActionText: string;
-  contentMd: string;
   descriptionMd: string;
+  summaryDescriptionMd?: string;
+  bodyHeading?: string;
```

Comment:
[sand] If we do opt to make this required in the CMS, it should be required here as well. I'm going to leave this as one comment for the other instances.